packages/components/app/styles/components/copy/button.scss

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// COPY > BUTTON
+//
+
+// Colors + States
+
+.hds-button--color-secondary {
```

Comment:
I would add a new class name to this to scope the styles instead of reusing the ".hds-button--color-secondary" class name.

Reply:
oh good point, I'll revert this change.